tensorboard/components/tf_categorization_utils/tf-tag-filterer.html

```diff
@@ -48,15 +48,28 @@
         tagFilter: {
           type: String,
           notify: true,
-          readOnly: true,
-          value: tf_storage.getStringInitializer('tagFilter',
-              {defaultValue: '', useLocalStorage: false}),
+          computed: '_setTagFilter(_tagFilter)',
+        },
+        _tagFilter: {
+          type: String,
+          value: tf_storage.getStringInitializer('tagFilter', {
+            defaultValue: '',
+            useLocalStorage: false,
+            polymerProperty: '_tagFilter',
+          }),
           observer: '_tagFilterObserver',
         },
       },
 
-      _tagFilterObserver: tf_storage.getStringObserver(
-          'tagFilter', {defaultValue: '', useLocalStorage: false}),
+      _tagFilterObserver: tf_storage.getStringObserver('tagFilter', {
+        defaultValue: '',
+        useLocalStorage: false,
+        polymerProperty: '_tagFilter',
+      }),
+
+      _setTagFilter() {
```

Comment:
I'd call this `_computeTagFilter()` for consistency with the other computed property methods, even though this one doesn't really do meaningful computation.  It's also pretty weird for a method named "setXX" to actually have no direct side effects (and which looks like a vanilla getter).